argoverse/evaluation/detection/utils.py

```diff
@@ -235,6 +254,16 @@ def filter_instances(
             filtered_annos = instances[dt_dists < max_detection_range]
     else:
         raise NotImplementedError("This filter metric is not implemented!")
+    
+    # ignore instances outside of region of interest (ROI) during evaluation
+    centers_cityframe = city_SE3_egovehicle.transform_point_cloud(centers_egoframe)
+    is_within_roi = avm.get_raster_layer_points_boolean(centers_cityframe, city_name, "roi")
```

Comment:
Should we consider using [remove_non_roi_points](https://github.com/argoai/argoverse-api/blob/a9abeb471d7c37aec98409668a232da2ebefa056/argoverse/map_representation/map_api.py#L346)?

Reply:
only thing is i think we want to get boolean information to filter out array items. remove_non_roi_points will give us back the relevant object centers, but it wont let us go backwards to determine which instances to select

Reply:
Ah, I see. Sounds good.